scipy/spatial/transform/_rotation.pyx

```diff
@@ -1001,11 +1001,10 @@ cdef class Rotation:
         for ind in range(num_rotations):
             angle = _norm3(crotvec[ind, :])
 
-            if angle <= 1e-3:  # small angle
-                angle2 = angle * angle
-                scale = 0.5 - angle2 / 48 + angle2 * angle2 / 3840
-            else:  # large angle
+            if angle > 1e-8:  # large angle
                 scale = sin(angle / 2) / angle
+            else:  # small angle
+                scale = 0.5
```

Comment:
Can you clarify this change (as in *why* this new value is computed this way? Also valid below. Since the reference is the same I'd just like to understand what changed.

Reply:
Hello @melissawm!
Here, the variable `scale` is computed with the three first terms of the Taylor series for $\frac{\sin(\theta / 2)}{\theta} \approx \frac{1}{2} - \frac{\theta^2}{48} + \frac{\theta^4}{3840}$. 
What I did is to use only the first term (which is a constant $\theta \approx \frac{1}{2}$) and compensate this by lowering the value for when this formula should be used.

In order to test this equivalence, you can run this:
```
def taylor_3terms(angle):
    angle2 = angle * angle
    return 0.5 - angle2 / 48 + angle2 * angle2 / 3840

def real(angle):
    return np.sin(angle / 2) / angle

def test(angle):
    error_3 = taylor_3terms(angle) - real(angle)
    error_1 = 0.5 - real(angle)
    print(f'Input: {angle}, Error Taylor 3 terms: {error_3}, Error Taylor 1 term: {error_1}')

test(1e-3)
test(1e-3)
test(1e-3)
```
And then run this:
```

```

Reply:
@evbernardes can you run the same analysis again but using mpmath as the ground truth?